Hotfix

VladEvka opened the pull request:
## Bugfix
1. [CLI-82] Local and global configs definition mismatch
2. [CLI-83] Encryption with the same recipient id-s in one request is possible
3. Fix name from 'key-gen' to 'keygen' in std::cerr

Commit:
[CLI-82] Local and global configs definition mismatch

Commit:
[CLI-83] Encryption with the same recipient id-s in one request is possible

Delete 'verbose' in 'readConfigFile'

Commit:
Fix name from 'key-gen' to keygen in std::cerr

Commit:
Exit from utility if unfound card[s]

Commit:
Add VIRGIL_CLI_VERSION_PRE_RELEASE - version feature, i.e. alpha, beta, rc1 etc

Commit:
Bump version number to 2.0.0-beta.1

Commit:
Add changes of version 2.0.0-beta.1 to the 'ChangeLog' file

Commit:
Switch branch from v2.0.0 to hotfix in JenkinsFile

SergeySeroshtan commented on ext/config_path_linux.h:
```diff
@@ -78,17 +88,7 @@ static inline std::string get_user_config_folder(const std::string& appname) {
  * @return get an absolute path, specific to this user.
  */
 static inline std::string get_all_user_config_folder(const std::string& appname) {
```
Function name and it's description are inconsistent.

SergeySeroshtan commented on ext/config_path_linux.h:
```diff
@@ -56,20 +56,30 @@
+ *
  * @return get an absolute path, for all users.
  */
 static inline std::string get_user_config_folder(const std::string& appname) {
```
Function name and it's description are inconsistent.

SergeySeroshtan commented on VERSION:
```diff
@@ -1 +1 @@
-2.0.0
+2.0.0-beta.1
```
Use name `2.0.0-beta1`

SergeySeroshtan commented on JenkinsFile:
```diff
@@ -2,7 +2,7 @@ stage 'VCS checkout'
 node('master'){
 	deleteDir()
-  git branch: 'v2.0.0', url: 'https://github.com/VirgilSecurity/virgil-cli.git'
+  git branch: 'hotfix', url: 'https://github.com/VirgilSecurity/virgil-cli.git'
```
Jenkisfile SHOULD NOT contain specific branch.

SergeySeroshtan commented on ChangeLog:
```diff
@@ -1,6 +1,14 @@
 virgil CLI ChangeLog (Sorted per date)
 
-= Version v2.0.0 released 2016-06-06
+= Version v2.0.0-beta.1 released 2016-06-28
```
Change version to v2.0.0-beta1

Commit:
Fix description for functions:

<linux, unix, apple>
get_user_config_folder()
get_all_user_config_folder()

because function name and it's description are inconsistent.

Commit:
Replace version from v2.0.0-beta.1 to v2.0.0-beta1

Commit:
Delete specific branch - hotfix in JenkinsFile, set branch - master.

SergeySeroshtan commented on JenkinsFile:
```diff
@@ -2,7 +2,7 @@ stage 'VCS checkout'
 node('master'){
 	deleteDir()
-  git branch: 'hotfix', url: 'https://github.com/VirgilSecurity/virgil-cli.git'
+  git branch: 'master', url: 'https://github.com/VirgilSecurity/virgil-cli.git'
```
Pay attention!!!! **NO specific** branch!!!

Commit:
Delete specific branch in JenkinsFile

Commit:
[CI] Fix install deps for Mac

Commit:
[CI] Delete build for Mac OS X
